Project EFT/Database/DBConnector.cs

```diff
@@ -239,6 +286,22 @@ public static Problem GetProblemByID(int ID)
             
         }
 
+        //this would be much better if the AnswerSubmission had a ProblemID in it as well...should be editted later
+        public static bool GetProblemCorrectValueByUserAndProblemID(int userID, int problemID)
+        {
+            MySqlCommand command = MakeCommand("SELECT AnswerSubmissions_IsCorrect FROM AnswerSubmissions WHERE User_ID = @user_id AND AnswerSubmissions_ProblemID = @problem_id AND AnswerSubmissions_IsCorrect = @correct");
+            command.Parameters.AddWithValue("@user_id", userID);
+            command.Parameters.AddWithValue("@problem_id", problemID);
+            command.Parameters.AddWithValue("@correct", true);
```

Comment:
can this be part of the statement, since it always takes the same value?

Reply:
Yes it can.